docs/section5/Rule5-36.md

```diff
@@ -0,0 +1,57 @@
+
+# Envelope - Rule 5-36  
+
+**Rule ID:** 5-36  
+**Rule Description:**  Skylight area must be allocated to surfaces in the same proportion in the baseline as in the proposed design; Skylight orientation and tilt shall be the same as in the proposed design.
+**Rule Assertion:** B-RMR (subsurface.glazed_area+subsurface.opaque_area) = expected value for each zone  
```

Comment:
@xingcx i think this rule description should not include the rule for skylight orientation and tilt. That should be a separate rule. Do we have a separate rule for that?